### DISCUSSION TITLE
How do I get calendar to behave consistently?

### DISCUSSION THREAD

**Discussion by designosis** [fomantic/Fomantic-UI#3130] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
Clicking into the calendar input opens the calendar.
Tabbing into it does not.
This fiddle shows the issue ... https://jsfiddle.net/p73ews80/1/
How does one open the calendar popup when the input gains focus?

**Comment by designosis** [comment]
Edit: Ok. Solution is to add `on: 'focus'` to the calendar params. This is not in the docs, just guessed. Perhaps the docs should include acceptable values for the `on` behavior.

**Comment by lubber-de** [comment, marked as answer]
It is documented, but not clear enough, so i enhanced this by https://github.com/fomantic/Fomantic-UI-Docs/pull/520
